Lib/pathlib.py

```diff
@@ -939,11 +939,17 @@ def match(self, path_pattern):
 
 
 class PurePosixPath(PurePath):
+    """On a POSIX system, instantiating a PurePath should return this object.
+    However, you can also instantiate it directly on any system.
+    """
```

Comment:
Thanks for the patch!

In the stdlib we try to follow the guidelines laid out in PEP 257. Mainly, the first line of a docstring should be a very short description that stands alone, and more information follows after a blank line (this is very similar to the recommendations for git commit messages).

Taking inspiration from the rst docs, here the first line could be something like `PurePath subclass for non-Windows systems.`, followed by a blank line then your original text.